docs/support/Troubleshooting/ticking-entities.mdx

```diff
@@ -31,7 +31,7 @@ If you are unable to find these options as detailed in the steps below, we recom
         - For ticking tile entities do the following:
             - Find the line `B:removeErroringTileEntities=false` and change it to `B:removeErroringTileEntities=true`
     </TabItem>
-    <TabItem value="forge-1.14-newer" label="Forge 1.14 and newer">
+    <TabItem value="forge-1.14-1.20.1" label="Forge 1.14 to Forge/Neoforge 1.20.1">
```

Comment:
Is this version range valid? Does normal Forge no longer use this? If it does, it should likely be 
```suggestion
    <TabItem value="forge-1.14-latest" label="Forge 1.14 to Forge Latest">
```